### PR TITLE
Support import of existing Forseti installations

### LINKED ISSUE #197: Allow module to import existing Forseti installations
Right now the user cannot import an existing installation using `terraform import`. This is due to the module adding random suffix to resources. It would be beneficial if the user could specify the resource names to the names of an existing installation, thus allowing for imports.

### PR DESCRIPTION
**Pull request by aaron-lane**
This branch adds support for importing existing Forseti installations. It adds variables to override the random resource name suffix and toggle rule management.

Fixes #197.

### CODE REVIEW

**Comment by gkowalski-google** [comment]
Hi @aaron-lane, thanks for working on this change, this will be nice to have for users when the python installer is deprecated. How much work is left on this change? Will this be ready for the release next week?

**Comment by kevensen** [comment]
@aaron-lane Please note that I am currently working on sub-moduliarizing the server components (#208) during this sprint starting with #256.  I do not know of the impact yet.

Is it possible to wait to merge this until after the dust settles?

**Comment by aaron-lane** [comment]
@gkowalski-google sorry for the delay. This branch is ready now. It is paired with https://github.com/forseti-security/forseti-security/pull/3166.

@kevensen it's intended to be released before the Python installer is declared obsolete and removed from the site. I have no opinion on deferring it, if the core team is comfortable doing so.

**Comment by morgante** [comment]
I'd suggest we prioritize testing/merging this first.

**Comment by kevensen** [comment]
@morgante @aaron-lane Can we get this tested and merged in the next 24 hours?  I am determined to get the sub-modularization work done this sprint.  That work is necessary for getting the on-GKE feature into beta.

**Comment by aaron-lane** [comment]
I successfully imported a deployment created with the Python installer. Does any one else have bandwidth to also test this?

File: helpers/import.sh [hunk 1]
```diff
@@ -0,0 +1,170 @@
...
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_compute_firewall.forseti-server-deny-all" "$PROJECT_ID/forseti-server-deny-all-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_compute_firewall.forseti-server-ssh-external" "$PROJECT_ID/forseti-server-allow-ssh-external-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_compute_firewall.forseti-server-allow-grpc" "$PROJECT_ID/forseti-server-allow-grpc-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_storage_bucket.server_config" "$PROJECT_ID/forseti-server-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_storage_bucket.cai_export" "$PROJECT_ID/forseti-cai-export-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_compute_instance.forseti-server" "$PROJECT_ID/us-central1-c/forseti-server-vm-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_sql_database_instance.master" "$PROJECT_ID/forseti-server-db-$RESOURCE_NAME_SUFFIX"
```
Comment (hshin-g) [inline helpers/import.sh]: Will need to change these to the cloudsql sub module.

**Comment by gkowalski-google** [comment]
> I successfully imported a deployment created with the Python installer. Does any one else have bandwidth to also test this?

I can help test this as well.

File: helpers/import.sh (hunk 1 above)
Comment (gkowalski-google) [reply helpers/import.sh]: Good catch @hshin-g! I have updated these.

File: helpers/import.sh [hunk 2]
```diff
@@ -0,0 +1,170 @@
...
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_compute_firewall.forseti-server-allow-grpc" "$PROJECT_ID/forseti-server-allow-grpc-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_storage_bucket.server_config" "$PROJECT_ID/forseti-server-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_storage_bucket.cai_export" "$PROJECT_ID/forseti-cai-export-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.server.google_compute_instance.forseti-server" "$PROJECT_ID/us-central1-c/forseti-server-vm-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.cloudsql.google_sql_database_instance.master" "$PROJECT_ID/forseti-server-db-$RESOURCE_NAME_SUFFIX"
+terraform import "module.$MODULE_LOCAL_NAME.module.cloudsql.google_sql_database.forseti-db" "$PROJECT_ID/forseti-server-db-$RESOURCE_NAME_SUFFIX/forseti_security"
+terraform import "module.$MODULE_LOCAL_NAME.module.cloudsql.google_sql_user.root" "$PROJECT_ID/forseti-server-db-$RESOURCE_NAME_SUFFIX/%/root"
```
Comment (hshin-g) [inline helpers/import.sh]: Should add in check for shared VPC here

Comment (aaron-lane) [reply helpers/import.sh]: Working on it presently. 👍

File: helpers/import.sh [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+  case ${opt} in
+    h )
+      show_help
+      exit 0
+      ;;
+    m )
+      MODULE_LOCAL_NAME="$OPTARG"
+      ;;
+    n )
+      NETWORK_PROJECT_ID="$OPTARG"
```
Comment (hshin-g) [inline helpers/import.sh]: Add `;;` here

**Pull request merged**